.github/workflows/deploy-standalone-plugins.yml

```diff
@@ -1,99 +1,201 @@
         uses: 10up/action-wordpress-plugin-deploy@stable
         with:
-          dry-run: ${{ matrix.dry-run }}
+          dry-run: ${{ github.event_name == 'workflow_dispatch' && inputs.dry-run || false }}
         env:
-          SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
+          SLUG: ${{ matrix.plugin }}
+          VERSION: ${{ env.PLUGIN_VERSION }}
           SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
-          SLUG: ${{ matrix.slug }}
-          VERSION: ${{ steps.set_version.outputs.version }}
-          BUILD_DIR: ./${{ matrix.directory }}/${{ matrix.slug }}
-          ASSETS_DIR: ./${{ matrix.directory }}/${{ matrix.slug }}/.wordpress-org
+          SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
+          BUILD_DIR: ./build/${{ matrix.plugin }}
+          ASSETS_DIR: ./plugins/${{ matrix.plugin }}/.wordpress-org
+
+      - name: Finish deployment
+        if: ${{ steps.wporg-deployment.outputs.deployment_id && always() }}
+        uses: bobheadxi/deployments@v1
+        with:
+          step: finish
+          token: ${{ secrets.GITHUB_TOKEN }}
+          status: ${{ job.status }}
+          deployment_id: ${{ steps.wporg-deployment.outputs.deployment_id }}
+          env: "wp.org plugin: ${{ matrix.plugin }}"
+          env_url: "https://wordpress.org/plugins/${{ matrix.plugin }}/"
+
+  release-assets:
+    name: Add release assets
+    needs: [ pre-run, deploy ]
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: write
+    if: github.event_name == 'release'
+    strategy:
+      matrix:
+        plugin: ${{ fromJSON(needs.pre-run.outputs.matrix) }}
+    steps:
+      - name: Check artifact existence
+        id: artifact-existence
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const getArtifact = await github.request('GET /repos/{owner}/{repo}/actions/runs/{run_id}/artifacts{?name}', {
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              run_id: context.runId,
+              name: "${{ matrix.plugin }}"
+            });
+
+            if (getArtifact.status !== 200) {
+              throw new Error(`Invalid response from GitHub API: ${getArtifact.status}`);
+            }
+
+            core.setOutput('exists', getArtifact.data.artifacts.length > 0);
+            core.debug(`Artifact for ${{ matrix.plugin }} exists: ${core.getInput('exists')} ? "true" : "false"`);
+
+      - name: Download artifact
+        if: steps.artifact-existence.outputs.exists == 'true'
+        uses: actions/download-artifact@v4
+        with:
+          name: ${{ matrix.plugin }}
+          path: ./build/dist
+
+      - name: Upload release assets
+        if: steps.artifact-existence.outputs.exists == 'true'
+        uses: softprops/action-gh-release@v1
+        with:
+          files: |
+            ./build/dist/${{ matrix.plugin }}.zip
+            ./build/dist/${{ matrix.plugin }}.zip.sha256
```

Comment:
So this will attach all standalone plugin ZIPs to the Performance Lab plugin release? If so, it seems like a missing piece in general for the standalone plugin story is how to trigger a release of a standalone plugin without having to do a release of Performance Lab as well. I guess that can still be done by [manually invoking the workflow](https://github.com/WordPress/performance/actions/workflows/deploy-standalone-plugins.yml), but then there would be no record of the standalone plugin release among the GitHub releases.

Reply:
Yes, that's right. For now, there are two scenarios:
- Release trigger.
- On workflow dispatch.

In the second case, there will be no release record as mentioned by @westonruter. In the last Performance chat, we discussed opening a new issue which I have just opened https://github.com/WordPress/performance/issues/1061 to discuss it.

Reply:
@westonruter Correct, at the moment the manual workflow is the way to do it, if we need to release something apart from a PL release.

A discussion on how to better address that is long overdue and has come up in a few places before. Let's tackle that in a separate issue. Potentially we can just create GitHub releases for the individual plugins and have the workflows adjust based on e.g. the release name or something like that what gets deployed.

Reply:
> So this will attach all standalone plugin ZIPs to the Performance Lab plugin release?

Only for such plugins that are not already released on WPOrg. See https://github.com/WordPress/performance/pull/1033/commits/0d5451d8bcd9c4619a1ccce54bf7e6e981b5987c